### ISSUE TITLE
Support list items with more than one line of text

### ISSUE THREAD

**Issue by telmiger** [tgrosinger/tw5-checklist#24]
At the moment, the second line of text is not indented. I tried fixing it, but ran into problems positioning the delete-task-cross (why is this in a div at the end of the line?).

**Comment by telmiger** [comment]
Never mind. It’s fixed for my application. See http://tid.li/tw5/hacks.html#Checklist-CSS

**Comment by tgrosinger** [comment]
The div at the end is for the sometimes-hidden "x" button to delete that item.

Your change (reproduced below) did not work for me when testing on my wiki. Perhaps you have other CSS rules which are also affecting this checklist? The spacing on yours definitely looks different from the stock example.

```css
.checklist span.checklistitem-body {
    margin-left: 0.87em;
}
```

**Comment by telmiger** [comment]
Solved in version 0.0.14.
